internal/cli/usercmd/app.go

```diff
@@ -201,6 +204,59 @@ func (c *EpinioClient) AppShow(appName string) error {
 	return c.printReplicaDetails(app)
 }
 
+// AppExport saves the named app, in the targeted namespace, to the directory.
+func (c *EpinioClient) AppExport(appName string, directory string) error {
+	log := c.Log.WithName("Apps").WithValues("Namespace", c.Settings.Namespace, "Application", appName)
+	log.Info("start")
+	defer log.Info("return")
+	details := log.V(1) // NOTE: Increment of level, not absolute.
+
+	c.ui.Note().
+		WithStringValue("Namespace", c.Settings.Namespace).
+		WithStringValue("Application", appName).
+		WithStringValue("Target Directory", directory).
+		Msg("Export application")
+
+	if err := c.TargetOk(); err != nil {
+		return err
+	}
+
+	details.Info("export application")
+
+	// 1. Create directory
+	// 2. Retrieve values.yaml for application
+	// 3. Retrieve chart tarball for application
+	// (4.) Retrieve application image (staged sources)
+	// Save all to the directory (in each step)
+	//
+	// While the `values.yaml` is likely quite small it is
+	// essentially a file like all the other parts (chart tarball,
+	// image). For consistency just handle all of them as files,
+	// in the same manner (web socket ?)
```

Comment:
```suggestion
```

The code is rather explanatory itself (like it should). I don't think we need to maintain comments as well.

Reply:
Done with commit 0f85444f1587cd6e32799c0ac2b12c093be2bd86